app/src/hooks/connectedWeb3.tsx

```diff
@@ -40,7 +43,10 @@ export const ConnectedWeb3: React.FC = props => {
   const [networkId, setNetworkId] = useState<number | null>(null)
   const safeAppInfo = useSafeApp()
   const context = useWeb3Context()
+
   const { account, active, error, library } = context
+  const [relay, setRelay] = useState(true)
+  const toggleRelay = () => setRelay(!relay)
```

Comment:
Also with this pr there will be no option to see mainnet markets? I see that we have toogleRelay but its not used anywhere..

Reply:
Yep there is no toggle UI yet we need to add this later 👍